src/scss/components/well/_well.scss

```diff
@@ -2,25 +2,25 @@
 // - - - - - - - - - - - - - - - - - - - - - - - - -
 
 section.ui-well {
-    background: $grey-900;
-    color: $white;
-    min-height: x-config-get('ui.well-min-height');
-    margin-bottom: $growcss-margin-default;
-    margin-top: $growcss-margin-default;
-    box-shadow: 0 0 rem(28) 0 rgba(102, 102, 102, 0.19) inset;
+  background: $grey-900;
+  color: $white;
```

Comment:
Properties should be ordered background, box-shadow, color, margin-bottom, margin-top, min-height